bucket_33/glew/specification

```diff
@@ -14,7 +14,7 @@ DOWNLOAD_GROUPS=	main
 SITES[main]=		SF/glew/glew/${PORTVERSION}
 DISTFILE[1]=		glew-${PORTVERSION}.tgz:main
 
-SPKGS[standard]=	complete primary docs
+SPKGS[standard]=	complete dev primary docs
```

Comment:
The order is important.  Primary *always* has to come after "complete"

Reply:
Change has been made. Now just working through the fallout. Looks like it is just libreoffice and one other library.